bundle/src/files.rs

```diff
@@ -45,125 +151,144 @@ pub struct FileSet {
 }
 
 impl FileSet {
-    /// Scan a file set from a glob path.
-    /// And generates file set using file counter.
-    ///
-    pub fn scan_from_glob(
-        repo_root: &str,
-        glob_with_status: JunitReportFileWithStatus,
```

Comment:
I think the change that you added removes the work done in https://github.com/trunk-io/analytics-cli/pull/242/files#diff-5fba23e112b0ab064ebf51f2e6ce7e7454395e6046cd7142b14f2fe0c8591e0eR157 to populate `resolved_status` (i.e. when coming from BEP) and use it as an override when determining failed_tests.

Reply:
Isn't this that exact logic?
https://github.com/trunk-io/analytics-cli/pull/276/files#diff-5fba23e112b0ab064ebf51f2e6ce7e7454395e6046cd7142b14f2fe0c8591e0eR101-R105

Reply:
Ah that's it. I can't read, was looking for `resolved_status` instead of `status` 🤦